src/modules/dashboard/sagas/motions/moveFundsMotion.ts

```diff
@@ -97,18 +98,22 @@ function* moveFundsMotion({
       'annotateMoveFundsMotion',
     ]);
 
-    // eslint-disable-next-line max-len
-    const encodedAction = colonyClient.interface.functions.moveFundsBetweenPots.encode(
-      [
-        permissionDomainId,
-        fromChildSkillIndex,
-        toChildSkillIndex,
-        fromPot,
-        toPot,
-        amount,
-        tokenAddress,
-      ],
-    );
+    const isOldVersion = parseInt(version, 10) <= 6;
```

Comment:
@chinins can you please change it to ColonyVersion.CeruleanLightweightSpaceship instead of fixed 6 and commit again? and then @ArmandoGraterol  if you could cherry pick again

Reply:
Sure, whenever the commit of @chinins is ready I'll cherry-pick it